packages/page/src/Infrastructure/Sulu/Admin/PageAdmin.php

```diff
@@ -299,50 +300,38 @@ public function configureViews(ViewCollection $viewCollection): void
                     ->setParent(static::EDIT_FORM_VIEW)
             );
 
-            if ($this->activityViewBuilderFactory->hasActivityListPermission()) {
-                $activityResourceTabViewName = PageAdmin::EDIT_FORM_VIEW . '.activity';
+            if ($this->activityViewBuilderFactory->hasActivityListPermission() || $this->contentVersionViewBuilderFactory->hasContentVersionListPermission()) {
```

Comment:
as discussed i would create the insight and version and in future reference tab should be created by the content view builder directly. so we not require content version view builder.